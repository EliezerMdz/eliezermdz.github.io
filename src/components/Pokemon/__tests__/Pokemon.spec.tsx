import { render } from '@testing-library/react';
import { Pokemon } from '..';
import { pokemons } from '../../../mocks';

it('renders', () => {
    const { container } = render(
        <Pokemon pokemon={pokemons[0]} />
    );
    expect(container).toMatchSnapshot();
});
