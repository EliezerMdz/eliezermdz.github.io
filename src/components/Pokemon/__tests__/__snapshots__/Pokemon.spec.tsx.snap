// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="card"
  >
    <div
      class="banner"
    />
    <img
      alt="Pkmn"
      class="pkmnImg"
      src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/1.png"
    />
    <h1
      class="name"
    >
      Bulbasaur
    </h1>
  </div>
</div>
`;
